.github/workflows/multi-arch-ghcr.yaml

```diff
@@ -0,0 +1,105 @@
+name: Create and publish a Docker multi-arch image to ghcr
+
+on:
+  push:
+    tags:        
+      - v0.*
```

Comment:
what would happen when we have a 1.* ?

Reply:
fyi please make the yaml file working under yamllint

Reply:
> what would happen when we have a 1.* ?

update it with the v* now

Reply:
> fyi please make the yaml file working under yamllint

have fixed now